anthropic/api.py

```diff
@@ -273,10 +273,13 @@ def _validate_request(params: dict) -> None:
 
 def _validate_prompt_length(params: dict) -> None:
     prompt: str = params["prompt"]
-    prompt_tokens = tokenizer.count_tokens(prompt)
-    max_tokens_to_sample: int = params["max_tokens_to_sample"]
-    token_limit = 9 * 1024
-    if prompt_tokens + max_tokens_to_sample > token_limit:
-        raise ApiException(
-            f"Prompt tokens ({prompt_tokens}) + max-sampled tokens ({max_tokens_to_sample}) exceeds max ({token_limit})",
-        )
+    try:
+        prompt_tokens = tokenizer.count_tokens(prompt)
+        max_tokens_to_sample: int = params["max_tokens_to_sample"]
+        token_limit = 9 * 1024
+        if prompt_tokens + max_tokens_to_sample > token_limit:
+            raise ApiException(
+                f"Prompt tokens ({prompt_tokens}) + max-sampled tokens ({max_tokens_to_sample}) exceeds max ({token_limit})",
+            )
+    except tokenizer.TokenizerException:
+        pass
```

Comment:
Does it make sense to log a warning or something here, just so that downstream users are aware that something has gone wrong and/or we get a signal?

Reply:
good idea, adding